src/DetailsView/components/assessment-instance-edit-and-remove-control.tsx

```diff
@@ -13,9 +13,11 @@ export interface AssessmentInstanceEditAndRemoveControlProps {
     test: VisualizationType;
     step: string;
     id: string;
-    description: string;
+    description?: string;
```

Comment:
Similarly, this component could instead consume that model instead of these multiple properties.

Reply:
propagated up